### PR TITLE
Update parse strings with split

### LINKED ISSUE #3330: Update How to Parse strings using Split
[This article](https://docs.microsoft.com/dotnet/csharp/programming-guide/strings/how-to-parse-strings-using-string-split) should be updated by:
- [x] update the title
- [x] link from C# Hello World Quick start
- [x] redirection
- [x] Better heading than "Example"
- [x] Split and LINQ example.

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #3330 

Update samples based on common questions, make samples interactive, and update links.

The referenced sample is now linked in the topic as well.

[Internal review link](https://review.docs.microsoft.com/en-us/dotnet/csharp/how-to/parse-strings-using-split?branch=pr-en-us-4083)

### CODE REVIEW

File: docs/csharp/how-to/parse-strings-using-split.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+This behvior makes it easier for formats like comma separated values (CSV)
+files representing tabular data. Consecutive commas represent a blank column.
+
+You can pass an optional <xref:System.StringSplitOptions.RemoveEmptyEntries> parameter to
+exclude any empty strings in the output. For more complicated processing of the returned
+ecollection, you can use [LINQ](../programming-guide/concepts/linq/index.md) to manipulate
+the result sequence.    
+
+<xref:System.String.Split%2A?displayProperty=nameWithType> can use
+multiple separator characters. The following example uses spaces, commas, periods, colons, and tabs, all passed in an array containing these separating characters to <xref:System.String.Split%2A>.  Each word in the target string's sentence displays separately from the resulting array of strings.
```
Comment (rpetrusha) [inline docs/csharp/how-to/parse-strings-using-split.md]: comma after "separating characters"
I'm not quite sure what the last sentence means.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
I've left a number of comments, but this looks good, @BillWagner; the C# REPL really helps here.

**Comment by BillWagner** [comment]
I addressed all your comments @rpetrusha This is ready to go again.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
This looks good, @BillWagner. I'll merge now.

**Pull request merged**